Datapack/data/dvz/functions/zombies/evoker/evoker_summon.mcfunction

```diff
@@ -0,0 +1,30 @@
+execute as @s if entity @s[tag=evoker,scores={DVZ.evokersummon.cool=0}] at @s run summon pillager ~ ~ ~ {Team:"zMONSTER",Tags:["legion"],CustomName:'{"text":"Illager Legionary"}',HandItems:[{id:"minecraft:crossbow",Count:1b,tag:{Enchantments:[{id:"minecraft:unbreaking",lvl:1s},{id:"minecraft:multishot",lvl:1s},{id:"minecraft:quick_charge",lvl:1s}]}},{id:"minecraft:tipped_arrow",Count:64b,tag:{CustomPotionEffects:[{Id:7,Amplifier:0b,Duration:10},{Id:20,Amplifier:1b,Duration:60}],CustomPotionColor:0}}],HandDropChances:[0.085F,0.000F],ArmorItems:[{},{},{},{id:"minecraft:black_stained_glass",Count:1b}],ArmorDropChances:[0.085F,0.085F,0.085F,0.085F],Attributes:[{Name:generic.movement_speed,Base:0.26},{Name:generic.attack_damage,Base:4},{Name:generic.armor,Base:12}]}
+execute as @s if entity @s[tag=evoker,scores={DVZ.evokersummon.cool=0}] at @s run summon pillager ~ ~ ~ {Team:"zMONSTER",Tags:["legion"],CustomName:'{"text":"Illager Legionary"}',HandItems:[{id:"minecraft:crossbow",Count:1b,tag:{Enchantments:[{id:"minecraft:unbreaking",lvl:1s},{id:"minecraft:multishot",lvl:1s},{id:"minecraft:quick_charge",lvl:1s}]}},{id:"minecraft:tipped_arrow",Count:64b,tag:{CustomPotionEffects:[{Id:7,Amplifier:0b,Duration:10},{Id:20,Amplifier:1b,Duration:60}],CustomPotionColor:0}}],HandDropChances:[0.085F,0.000F],ArmorItems:[{},{},{},{id:"minecraft:black_stained_glass",Count:1b}],ArmorDropChances:[0.085F,0.085F,0.085F,0.085F],Attributes:[{Name:generic.movement_speed,Base:0.26},{Name:generic.attack_damage,Base:4},{Name:generic.armor,Base:12}]}
+execute as @s if entity @s[tag=evoker,scores={DVZ.evokersummon.cool=0}] at @s run summon pillager ~ ~ ~ {Team:"zMONSTER",Tags:["legion"],CustomName:'{"text":"Illager Legionary"}',HandItems:[{id:"minecraft:crossbow",Count:1b,tag:{Enchantments:[{id:"minecraft:unbreaking",lvl:1s},{id:"minecraft:multishot",lvl:1s},{id:"minecraft:quick_charge",lvl:1s}]}},{id:"minecraft:tipped_arrow",Count:64b,tag:{CustomPotionEffects:[{Id:7,Amplifier:0b,Duration:10},{Id:20,Amplifier:1b,Duration:60}],CustomPotionColor:0}}],HandDropChances:[0.085F,0.000F],ArmorItems:[{},{},{},{id:"minecraft:black_stained_glass",Count:1b}],ArmorDropChances:[0.085F,0.085F,0.085F,0.085F],Attributes:[{Name:generic.movement_speed,Base:0.26},{Name:generic.attack_damage,Base:4},{Name:generic.armor,Base:12}]}
+
+execute as @s if entity @s[tag=evoker,scores={DVZ.evokersummon.cool=0}] run scoreboard players set @s DVZ.evokersummon.cool 25
```

Comment:
An item this powerful should probably have only 1 use.

Reply:
Sort of core identity of the class, increasing cooldown on all abilities for a nerf but will see if summons needs a nerf depending on how they work in combat